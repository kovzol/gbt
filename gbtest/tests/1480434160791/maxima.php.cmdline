#!/bin/sh
cd tests/1480434160791
cat maxima.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/maxima > maxima.php.out 2>maxima.php.err