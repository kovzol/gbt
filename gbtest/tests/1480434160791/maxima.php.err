0.65user 0.02system 0:00.69elapsed 98%CPU (0avgtext+0avgdata 228376maxresident)k
0inputs+480outputs (0major+4835minor)pagefaults 0swaps
