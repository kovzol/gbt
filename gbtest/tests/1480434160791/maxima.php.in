load(grobner)$ poly_monomial_order:grevlex$ poly_reduction(poly_grobner([-1*v9,-1*v7,-1+v10,v8,-1+v5,v3,v6,v4,v25^2*v24^2+-1*v26^2*v23^2,-1*v18+v16+v5+-1*v3,-1*v17+v15+-1*v6+v4,v19*v18+-1*v20*v17+-1*v19*v16+v17*v16+v20*v15+-1*v18*v15,-1*v6+v4+-1*v5*v4+v6*v3,-1*v19*v6+v20*v5+v19*v4+-1*v5*v4+-1*v20*v3+v6*v3,v24^2+-1*v16^2+-1*v15^2+2*v16*v14+-1*v14^2+2*v15*v13+-1*v13^2,-1+v25^2+-1*v20^2+2*v19+-1*v19^2,-1*v5*v4+v6*v3+v5*v2+-1*v3*v2+-1*v6*v1+v4*v1,v23^2+-1*v16^2+-1*v15^2+2*v16*v2+-1*v2^2+2*v15*v1+-1*v1^2,-1*v9*v8+v10*v7+v9*v2+-1*v7*v2+-1*v10*v1+v8*v1,v26^2+-1*v20^2+-1*v19^2+2*v20*v2+-1*v2^2+2*v19*v1+-1*v1^2,1+-1*v16^2+-1*v15^2+2*v16*v2+-2*v1+2*v15*v1,-1*v15*v14+v16*v13+v15*v2+-1*v13*v2+-1*v16*v1+v14*v1],[v26,v24,v25,v23,v20,v18,v19,v16,v17,v15,v10,v9,v8,v7,v6,v5,v4,v3,v2,v1,v13,v14]),[v26,v24,v25,v23,v20,v18,v19,v16,v17,v15,v10,v9,v8,v7,v6,v5,v4,v3,v2,v1,v13,v14]);