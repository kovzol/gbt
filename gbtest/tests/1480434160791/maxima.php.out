
Maxima 5.37.2 http://maxima.sourceforge.net
using Lisp GNU Common Lisp (GCL) GCL 2.6.12
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1) (%o3) [v9, v7, v10 - 1, v8, v5 - 1, v3, v6, v4, v20, v2, v15 - v19, v1, 
                                        2            2
v13 v16 - v14 v15, (- 2 v14 v16) - 2 v14  v15 - 2 v13  v15 - 4 v13 v15 - 2 v15
      2        2                                                   2      2
 + v14  + 2 v13  + 4 v13 + 1, (- 2 v14 v15 v16) + 2 v14 v16 + 2 v15  + v14  v15
            2                     2                      2
 + v15 - v14  - 1, 2 v15 v16 - v14  v16 - v16 + 2 v14 v15  - 2 v13 v14 v15
                                          3                         2
 - 4 v14 v15 + 2 v13 v14 + 2 v14, (- 2 v14  v16) + 2 v14 v16 + 2 v14  v15
                          4        2
 + 4 v13 v15 + 2 v15 + v14  - 4 v13  - 4 v13 - 1, 
                 4            2                                  4      4
8 v14 v16 + 2 v14  v15 + 8 v14  v15 + 8 v13 v15 + 6 v15 - v13 v14  + v14
            2        2        3        2
 - 2 v13 v14  - 6 v14  + 4 v13  - 4 v13  - 13 v13 - 3, 
   6      2    4        4        2    2      2        4        3        2
v14  + v13  v14  - 2 v14  - 2 v13  v14  + v14  - 4 v13  - 8 v13  - 3 v13 , 
                                    2                               2      2
(- v18) + v16 + 1, v15 - v17, (- v24 ) - 2 v14 v16 - 2 v13 v15 + v14  + v13
            2       2                     2     2      2        2       2
 + 1, (- v25 ) + v15  - 2 v15 + 1, 1 - v23 , v15  - v26 , (- v16 ) - v15  + 1, 
                     2        2      2                2
2 v14 v16 + 2 v13 v15  + 2 v15  + v14  v15 + v15 - v14  - 2 v13 - 1, 
                     2            2                              4            2
4 v14 v16 + 2 v13 v14  v15 + 6 v14  v15 + 2 v13 v15 + 2 v15 + v14  - 2 v13 v14
        2        2
 - 4 v14  - 4 v13  - 6 v13 - 1]
(%i4) 